projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html

```diff
@@ -47,42 +47,42 @@
                   aria-label="upvote the post" id="up-vote">
                   <!-- upward arrow -->
-                  <img src="./assets/discussion-ui/images/up-arrow.png" alt="up-arrow" width="14px" height="14px" class="icons">
+                  <img src="./assets/discussion-ui/images/up-arrow.png" alt="up-arrow" width="16px" height="16px" class="icons">
                 </a>
                 <a href="javascript:void(0)" i18n-aria-label *ngIf="post.upvoted" role="link" (click)="deleteVote(post);logTelemetry($event, post)"
                   aria-label="remove upvote" id="delete-vote">
                   <!-- upward arrow -->
-                  <img src="./assets/discussion-ui/images/select-up-arrow.png" alt="select-up-arrow" width="14px" height="14px">
+                  <img src="./assets/discussion-ui/images/up.svg" alt="select-up-arrow" width="16px" height="16px">
                 </a>
-                <span class="pad-4" role="text"
+                <span class="ml-8" role="text"
                   [attr.aria-label]="'total upvoat is' + data?.upvotes">{{data?.upvotes}}</span>
               </span>
-              <span class="flex margin-right-l mobile">
+              <span class="flex mr-24 mobile">
                 <a href="javascript:void(0)" *ngIf="!post?.downvoted" i18n-aria-label role="link"
                   (click)="downvote(post);logTelemetry($event, post)" aria-label="downvote post" id="down-vote">
                   <!-- downward arrow -->
-                  <img src="./assets/discussion-ui/images/down-arrow.png" width="14px" height="14px">
+                  <img src="./assets/discussion-ui/images/down-arrow.png" width="16px" height="16px">
                 </a>
                 <a href="javascript:void(0)" *ngIf="post?.downvoted" i18n-aria-label role="link"
                   (click)="deleteVote(post);logTelemetry($event, post)" aria-label="remove down vote" id="delete-vote">
                   <!-- downward arrow -->
-                  <img src="./assets/discussion-ui/images/select-down-arrow.png" width="14px" height="14px">
+                  <img src="./assets/discussion-ui/images/down.svg" width="16px" height="16px">
                 </a>
-                <span class="pad-4" role="text"
+                <span class="ml-8" role="text"
                   [attr.aria-label]="'total upvoat is' + data?.upvotes">{{data?.downvotes}}</span>
               </span>
-              <span class="flex margin-right-l mobile" role="link" tabindex="0">
+              <span class="flex mr-24 mobile" role="link" tabindex="0">
                  <!-- trending -->
-                 <img src="./assets/discussion-ui/images/profits.png" alt="profit-image" width="14px" height="14px">
-                <span class="pad-4" i18n-aria-label aria-label="views count">{{data.viewcount}} views</span>
+                 <img src="./assets/discussion-ui/images/views.svg" alt="profit-image" width="16px" height="16px">
```

Comment:
Why so many inline styles here for 
width & height of the image. 

The same code is duplicated for all the images. define the class & use the same class